packages/navigator/src/browser/navigator-contribution.ts

```diff
@@ -619,7 +620,7 @@ export class FileNavigatorContribution extends AbstractViewContribution<FileNavi
         if (this.workspacePreferences['workspace.supportMultiRootWorkspace']) {
             this.toDisposeAddRemoveFolderActions.push(registry.registerMenuAction(NavigatorContextMenu.WORKSPACE, {
                 commandId: FileNavigatorCommands.ADD_ROOT_FOLDER.id,
-                label: WorkspaceCommands.ADD_FOLDER.label!
+                label: WorkspaceCommands.ADD_FOLDER.label
```

Comment:
In several places (e.g. `packages/editor/src/browser/editor-menu.ts`), you've added the non-null `!` to labels for menu actions. Here you're removing it. Is there any particular reason to do either?

Reply:
Actually no. These are just left-overs of my previous changes, which I didn't clean up.